src/run_client.py

```diff
@@ -1,43 +1,46 @@
-#### ASYNC ####
 import asyncio
 
 from client import AgentClient
 from schema import ChatMessage
 
 
 async def amain() -> None:
+    #### ASYNC ####
     client = AgentClient()
 
     print("Chat example:")
-    response = await client.ainvoke("Tell me a brief joke?", model="llama-3.1-70b")
+    response = await client.ainvoke("Tell me a brief joke?", model="gpt-4o")
     response.pretty_print()
 
     print("\nStream example:")
     async for message in client.astream("Share a quick fun fact?"):
         if isinstance(message, str):
-            print(message, flush=True, end="|")
+            print(message, flush=True, end="")
         elif isinstance(message, ChatMessage):
             print("\n", flush=True)
             message.pretty_print()
         else:
             print(f"ERROR: Unknown type - {type(message)}")
 
 
-asyncio.run(amain())
+def main() -> None:
+    #### SYNC ####
+    client = AgentClient()
 
-#### SYNC ####
-client = AgentClient()
+    print("Chat example:")
+    response = client.invoke("Tell me a brief joke?", model="gpt-4o")
+    response.pretty_print()
+
+    print("\nStream example:")
+    for message in client.stream("Share a quick fun fact?"):
+        if isinstance(message, str):
+            print(message, flush=True, end="|")
+        elif isinstance(message, ChatMessage):
+            print("\n", flush=True)
+            message.pretty_print()
+        else:
+            print(f"ERROR: Unknown type - {type(message)}")
 
-print("Chat example:")
-response = client.invoke("Tell me a brief joke?", model="llama-3.1-70b")
-response.pretty_print()
 
-print("\nStream example:")
-for message in client.stream("Share a quick fun fact?"):
-    if isinstance(message, str):
-        print(message, flush=True, end="|")
-    elif isinstance(message, ChatMessage):
-        print("\n", flush=True)
-        message.pretty_print()
-    else:
-        print(f"ERROR: Unknown type - {type(message)}")
+if __name__ == "__main__":
+    asyncio.run(amain())
```

Comment:
need to run `main()` after this, right?

Reply:
It's not necessary; I just want to say that this file looks like a test entry file.
There shouldn't be any files importing this module, right?

Reply:
Correct it’s just for manual testing the API without Streamlit. not for importing